client/src/pages/Profile/index.js

```diff
@@ -1,183 +1,227 @@
-                        <p className = {classes.meetinginfo}>Meeting with XYZ on JavaScript</p>
-                    </div>                    
-            </div>
-        )
-    }
+class RecentMeet extends Component {
+  render() {
+    return (
+      <div className={classes.card}>
+        <h1 className={classes.cardtitle}>Recent Meetings</h1>
+        <div className={classes.meetings}>
+          <button>Endorse</button>
+          <p className={classes.meetinginfo}>Meeting with XYZ on JavaScript</p>
+        </div>
+      </div>
+    );
+  }
 }
 
 class Popup extends Component {
-    render(){
-        return(
-            <div className = {classes.popupoverlay}>
-                <div className = {classes.popupcontent}>
-                    <button className = {classes.close} onClick = {this.props.action}><icon>Close</icon></button>
-                    <form>
-                        <label for="start">Date:</label>
-                        <input type="date"></input>
-                        <label for="start">Time:</label>
-                        <input type="time"></input>
-                        <input type="submit"></input>
-                    </form>
-                    
-                </div>
-            </div>
-        )
-    }
+  render() {
+    return (
+      <div className={classes.popupoverlay}>
+        <div className={classes.popupcontent}>
+          <button className={classes.close} onClick={this.props.action}>
+            <icon>Close</icon>
+          </button>
+          <form>
+            <label for="start">Date:</label>
+            <input type="date"></input>
+            <label for="start">Time:</label>
+            <input type="time"></input>
+            <input type="submit"></input>
+          </form>
+        </div>
+      </div>
+    );
+  }
 }
 
 class MeetingContent extends Component {
-    constructor(props) {
-        super(props)
-        this.ResheduleInActive = this.ResheduleInActive.bind(this)
-        this.ResheduleActive = this.ResheduleActive.bind(this)
-        this.state = {showRegister:false}
-      }
-    ResheduleInActive(){
-        this.setState({...this.state,showRegister:false})
-    }
-    ResheduleActive(){
-        this.setState({...this.state,showRegister:true})
-    }
-    render(){
-        return(
-            <div id={classes.meeting} >
-                <section className = {classes.cards}>                
-                <UpcomingMeet action ={this.ResheduleActive}/>
-                <RecentMeet/>                     
-                </section>
-                {this.state.showRegister && <Popup action = {this.ResheduleInActive}/>}
-            </div>
-        )
-    }
+  constructor(props) {
+    super(props);
+    this.ResheduleInActive = this.ResheduleInActive.bind(this);
+    this.ResheduleActive = this.ResheduleActive.bind(this);
+    this.state = { showRegister: false };
+  }
+  ResheduleInActive() {
+    this.setState({ ...this.state, showRegister: false });
```

Comment:
Change this to 
```suggestion
    this.setState({showRegister: false });
```
React's `setState` ask only for those values you want to update